packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -139,6 +139,21 @@ const propTypes = {
    * Boolean indicating whether or not the DataGrid should hide the column headers.
    */
   hasVisibleColumnHeaders: PropTypes.bool,
+
+  /**
+   * @private
+   * A Unique Identifier of the [column](/components/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/about#column).
+   * If provided, column with specified identifier will be highlighted in data-grid.
+   *
+   * ![IMPORTANT](https://badgen.net/badge/UX/Design-Standards/blue) The column highlight feature should be limited specifically to
+   * time and timeline concepts only, best used with special instruction and guidance from User Experience to ensure proper standards.
+   */
+  columnHighlightId: PropTypes.string,
+
+  /**
+   * The color to be used for highlighting a column.
+   */
+  colorHighlightColor: PropTypes.oneOf(['orange', 'green']),
```

Comment:
Can this prop be renamed to `colHighlightColor` or `columnHighlightColor`?

Reply:
Fixed typo: https://github.com/cerner/terra-framework/pull/1985/commits/f316f91d6aa32289ef93a1a41ef44f72e62761e8#diff-64d05a135418b5eaa187214d1d0a5ed8df59d36a646018d75c2cf345b93a30e8